src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
+
+Visit the `IntelliJ download site <https://www.jetbrains.com/idea/download/>`__.
+Select your operating system and the Community version. Follow the installation
+prompts to select your settings. When you reach the window asking for your UI theme,
+you can choose to *Skip Remaining and Set Defaults*. You will finish on an IntelliJ
+window listing the options to *Create New Project*, *Import Project*, *Open*, and 
+*Check out from Version Control*.
+
+.. figure:: figures/IntelliJWelcome.png
+   :scale: 80%
+   :alt: Welcome window for IntelliJ
+
+   IntelliJ welcome window
+
+You've installed IntelliJ and you're ready to start exploring its many features.
+
+.. _create-new-java-project:
+
+Your First Java Project
+------------------------
+
+Following the "Hello, World" trend, let's create a new IntelliJ project.
+
+#. Create a new folder to hold your Java practice files. Since you will be
+   creating lots of small projects as you move through this course, we
+   suggest that you also add sub-folders with names corresponding to the
+   related chapters and projects. Something like
+   ``java-practice/chapter-name/project-name``.
+#. Select the *Create New Project* option from the welcome
+   screen.
+
+   .. figure:: figures/IntelliJWelcome.png
+      :scale: 80%
+      :alt: Welcome window for IntelliJ
+
+      Create new project
+
+#. Clicking *New Project* opens a window with a series of project settings to
+   select. For this first setting, make sure your selected project SDK is the JDK
+   you have installed. This allows IntelliJ to compile our Java code in-app. 
+   Click *Next* in the lower right corner of the window to continue selecting settings.
+
+   .. figure:: figures/projectSDK.png
+      :alt: Select project SDK
+
+      Select project SDK
+
+#. In the second window, select *Create project from template*. This gives us
+   some of the Java project scaffolding to save us some time with infrastructure. 
+
+   .. figure:: figures/projectTemplate.png
+      :alt: Select project template
+
+      Select project template
+
+#. On the next window, enter ``HelloWorld`` for the name of the project.
+   Click on the "3-dot" button to select a location to save the project. Here you can
+   choose you Java projects folder you created in step one. Leave the base package as
+   ``com.company``. 
+
+   .. figure:: figures/newProjectName.png
+      :alt: New project window for IntelliJ
+
+      Create the ``HelloWorld`` project in you Java projects folder.
+
+#. Click *Finish* to create the project. Below is the view of your new project:
+
+   .. figure:: figures/newProjectView.png
+      :alt: New project view
+
+      Initial IntelliJ project view
+
+   The section on the left is the project's file tree. Clicking the triangle next to 
+   the project name, ``HelloWorld``, displays the ``src`` file, followed by the base package
+   we created, and finally our ``Main.java`` file. ``Main.java`` is also opened on the right
```

Comment:
The Java naming convention has FileName.java = ClassName, so long as there is a ``main`` method.  Having a HelloWorld project with ``Main.java`` might appear to contradict the convention we emphasize later in the chapter.